lasagne/layers/conv.py

```diff
@@ -1047,88 +1352,47 @@ class TransposedConv3DLayer(BaseConvLayer):  # pragma: no cover
 
     Notes
     -----
-    The transposed convolution is implemented as the backward pass of a
-    corresponding non-transposed convolution. It can be thought of as dilating
-    the input (by adding ``stride - 1`` zeros between adjacent input elements),
-    padding it with ``filter_size - 1 - crop`` zeros, and cross-correlating it
-    with the filters. See [1]_ for more background.
+    See `BaseTransposedConvLayer` for more.
```

Comment:
Aha, I guess we will have to copy this note to all subclasses, as we cannot refer to the base class.